corehq/apps/domain/views.py

```diff
@@ -2255,10 +2243,6 @@ class EditInternalDomainInfoView(BaseInternalDomainSettingsView):
     def dispatch(self, request, *args, **kwargs):
         return super(BaseInternalDomainSettingsView, self).dispatch(request, *args, **kwargs)
 
-    @property
-    def autocomplete_fields(self):
```

Comment:
The countries input on this page isn't an autocomplete anymore, it's a multiselect.